website/templates/components/dashboard_templates.mako

```diff
@@ -138,6 +138,15 @@
             <!-- flashed validation message -->
             <span class="text-danger" data-bind="text: errorMessage"></span>
             <br />
+            
+            <label>Category </label>
+            <select class="form-control"
+                    data-bind="css: {disabled: usingTemplate},
```

Comment:
I'm still not sure that it makes sense to disable this input if a template is selected.